### ISSUE TITLE
[Accessibility] [Supporting the platform -Playground]: Page content is not visible properly when the page is zoomed to 400%. [Sev2][WCAG1.4.10]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#35]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Prerequisites:** Set the Display resolution to 1280*1024 and browser zoom to 400%

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘Playground’ link and activate it.
4.	Now set the display resolution to 1280*1024 and browser zoom to 400% and check whether the page content is properly visible or not.

**Actual:**
Page content is not visible properly when the page is zoomed to 400%.

**Expected:**
Page content should visible properly without loss of any content when the page is zoomed to 400%.

**User Impact:**
People with visual disabilities must be able to scale the browser up to 400% without losing the ability to read and track text. Two-dimensional scrolling impairs tracking and reading text, thus blocking the ability to understand or use the content.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/reflow

https://user-images.githubusercontent.com/88436727/150644717-1df386da-abc7-4871-a9c7-cba23d38bdf1.mp4

**Comment by apostolisms** [comment]
Fix here:https://github.com/microsoft/tsdoc/pull/334
